ember-toucan-core/src/components/form/textarea-field.hbs

```diff
@@ -0,0 +1,28 @@
+<div class="flex flex-col">
+  <Form::Field as |field|>
+    <field.Label for={{field.id}} data-textarea-label>{{@label}}</field.Label>
+
+    {{#if @hint}}
+      <field.Hint id={{field.hintId}} data-textarea-hint>{{@hint}}</field.Hint>
+    {{/if}}
+
+    <field.Control>
+      <textarea
```

Comment:
I am glad you brought this up! I was thinking the exact same but then I thought maybe I'm overthinking / over solving and decided not to do it. 

I agree though, I think it makes sense so that folks can only use those individual atoms as needed and they're styled as Toucan components.

I'll get these changes added! 🙇